exercises/practice/zebra-puzzle/zebra_puzzle.h

```diff
@@ -0,0 +1,10 @@
+#ifndef ZEBRA_PUZZLE_H
+#define ZEBRA_PUZZLE_H
+
+/// Determine the nationality of the resident who drinks water.
+const char *drinks_water(void);
+
+/// Determine the nationality of the resident who owns the zebra.
+const char *owns_zebra(void);
+
+#endif // ZEBRA_PUZZLE_H
```

Comment:
```suggestion
#endif
```

Removed for consistency with other exercises